src/aind_slims_api/types.py

```diff
@@ -0,0 +1,19 @@
+"""Common types for the SLIMS API.
+"""
+
+from typing import Literal
+
+# List of slims tables manually accessed, there are many more
+SLIMSTABLES = Literal[
```

Comment:
Maybe we can call it `SLIMS_TABLES`? Either way is probably fine.

Reply:
resolved in [af8b626](https://github.com/AllenNeuralDynamics/aind-slims-api/pull/20/commits/af8b626023a7ade048ae10f8ac599fd5fdeec661)